shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -45,14 +45,16 @@
     PointerDeviceKind.MOUSE,
     PointerDeviceKind.STYLUS,
     PointerDeviceKind.INVERTED_STYLUS,
+    PointerDeviceKind.TRACKPAD,
     PointerDeviceKind.UNKNOWN
   })
   private @interface PointerDeviceKind {
     int TOUCH = 0;
     int MOUSE = 1;
     int STYLUS = 2;
     int INVERTED_STYLUS = 3;
-    int UNKNOWN = 4;
+    int TRACKPAD = 4;
+    int UNKNOWN = 5;
```

Comment:
This smells bad, but it doesn't look like this will break anything. Nevertheless, I think it'd be better to assign UNKNOWN with some value like 99.

Reply:
Ok. Since other platforms are using 5, we should keep them consistent.